Lib/pdb.py

```diff
@@ -232,6 +232,8 @@ class Pdb(bdb.Bdb, cmd.Cmd):
     # but in case there are recursions, we stop at 999.
     MAX_CHAINED_EXCEPTION_DEPTH = 999
 
+    _file_mtime_table = {}
```

Comment:
should this be cleared when rerunning the program?

Reply:
Yeah probably, to keep the behavior consistent. Otherwise if somehow a user modified a lot of the files at the same time, the user might get warnings for each file after rerun.